ports/pthreadpool/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "pthreadpool",
+  "version-string": "2020-02-21",
```

```suggestion
  "version-date": "2020-02-21",
```
It seems more proper to use `version-date` field here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions